src/CabanaPD_Particles.hpp

```diff
@@ -796,14 +746,127 @@ class Particles<MemorySpace, PMB, TemperatureDependent, Dimension>
     }
 
     aosoa_temp_type _aosoa_temp;
+};
 
-#ifdef Cabana_ENABLE_HDF5
-    using base_type::h5_config;
-#endif
+template <class MemorySpace, class ModelType, class ThermalType, int Dimension>
+class Particles<MemorySpace, ModelType, ThermalType, OutputEnergy, Dimension>
```

Comment:
A renamed tag like `BaseOutput` for the base class makes this derived class definition make more sense.

Reply:
Agreed it's much more clear